File: LeetCode/0066. Plus One/C#/01. Plus One.cs
int[] PlusOne(int[] digits)
{
    BigInteger result = 1;
    string arrayValue = string.Empty;
    for (int x = 0; x < digits.Length; x++)
        arrayValue += digits[x];
    result += BigInteger.Parse(arrayValue);
    return result.ToString().Select(arrayNum => Convert.ToInt32(arrayNum) - 48).ToArray();
}

File: LeetCode/0066. Plus One/JavaScript/01. Plus One.js
var plusOne = function (digits) {
    for (let x = digits.length - 1; x >= 0; x--) {
        if (digits[x] < 9) {
            digits[x]++;
            return digits;
        }
        digits[x] = 0;
    }
    digits.length = digits.length + 1;
    digits[digits.length - 1] = 0;
    digits[0] = 1;
    return digits;
}